source/assets/stylesheets/components/_docs-nav.scss

```diff
@@ -11,17 +11,23 @@
       position: fixed;
       top: 0;
       max-height: 100vh;
-      overflow: -moz-scrollbars-none;
-      overflow-y: scroll;
-      overflow-x: visible;
+      overflow: hidden;
 
       &::-webkit-scrollbar {
         width: 0 !important;
       }
+
+      .docs-nav__inner {
+        overflow-y: scroll;
```

Comment:
Should this be `auto`? I believe using `scroll` will _always_ show scroll bars, whereas with `auto` they will only show if needed.